app/routes/__index/dns-records/new.tsx

```diff
@@ -47,15 +47,16 @@ export const action = async ({ request }: ActionArgs) => {
 
 export default function NewDnsRecordRoute() {
   const actionData = useActionData();
+  logger.info(actionData);
```

Comment:
What is this for?

Reply:
Sorry, I was thinking to add dynamic domain name for the user. Should be removed.